ci/ciimage/bionic/install.sh

```diff
@@ -58,6 +58,22 @@ update-alternatives --install /usr/bin/python3 python3 /usr/bin/python3.7 2 \
 
 python3 -m pip install -U "${base_python_pkgs[@]}" "${python_pkgs[@]}"
 
+pushd /opt
+# Download and install PyPy3.8, and link it to /usr/bin/pypy3
+# At some point it would be more robust to download and parse
+# https://downloads.python.org/pypy/versions.json
+wget https://downloads.python.org/pypy/pypy3.8-v7.3.9-linux64.tar.bz2
+pypy_sha256="08be25ec82fc5d23b78563eda144923517daba481a90af0ace7a047c9c9a3c34"
+if [ $pypy_sha256 != $(sha256sum pypy3.8-v7.3.9-linux64.tar.bz2 | cut -f1 -d" ") ]; then
+  echo bad sha256 for PyPy
+  exit -1
+fi
+tar -xf pypy3.8-v7.3.9-linux64.tar.bz2
+pypy3.8-v7.3.9-linux64/bin/pypy3 -m ensurepip
+popd
+ln -s /usr/bin/pypy3 /opt/pypy3.8-v7.3.9-linux64/bin/pypy3
```

Comment:
I am prone to forgetting how this works too. Here's what I came up with to remember:

The `ln` command is similar to the `cp` command. First you list the thing you want to install, in this case a string pointing to a destination, and then you list the location you want to install it *to* (by writing the first string as a symlink object).

So `ln -s foo bar` creates a symlink which *finds* foo, and the symlink is installed to bar.

` ln -s /usr/bin/pypy3 /opt/pypy3.8-v7.3.9-linux64/bin/pypy3` creates a symlink which *finds* /usr/bin/pypy3, and it installs the symlink to /opt/pypy3.8-v7.3.9-linux64/bin/pypy3

Reply:
Thanks. I added 5 lines and needed 4 commits to get them going :(. Sorry for the wasted time.

Reply:
No big deal, I've needed far more than just 4 commits for far fewer than 5 lines. Also, you did several of those commits while I wasn't looking so I didn't actually end up wasting my time (I guess github wasted their donated CI time but they waste much more than that on a regular basis).